.github/workflows/azure-function.yml

```diff
@@ -3,6 +3,11 @@ name: ♻ azure-function
 on:
   workflow_call:
     inputs:
+      runs_on:
+        description: The label of the runner (GitHub- or self-hosted) to run this workflow on. Defaults to 'ubuntu-22.04'.
```

Comment:
Replace with backticks to format as code:

```suggestion
        description: The label of the runner (GitHub- or self-hosted) to run this workflow on. Defaults to `ubuntu-22.04`.
```

See for example the tooltip in VS Code for the `backend_config` input in workflow `terraform.yml`, where we use this formatting for the example file name:

![bilde](https://github.com/user-attachments/assets/3fd4abe3-e78e-45c8-a2c8-7cd9540ee9c5)

Reply:
pretty safe to assume i should do the same with all the workflows right ?

Reply:
correct 😊